### PR TITLE
[stable/rabbitmq] Load Balancer sources

### PR DESCRIPTION
**Pull request by jmvizcainoio**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/rabbitmq/templates/svc.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ metadata:
 {{- end }}
 spec:
   type: {{ .Values.service.type }}
+{{- if .Values.service.loadBalancerSourceRanges }}
```
Comment (juan131) [inline stable/rabbitmq/templates/svc.yaml]: Shouldn't you check if the service.type is a **LoadBalancer** too?

```yaml
{{- and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerSourceRanges }}
```

**Review by juan131 — CHANGES_REQUESTED** [review]
@jmvizcainoio

File: stable/rabbitmq/values.yaml [hunk 2]
```diff
@@ -139,6 +139,11 @@ service:
   annotations:
     # service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
 
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  # loadBalancerSourceRanges:
+  #  - 10.10.10.0/24
+
```
Comment (carrodher) [inline stable/rabbitmq/values.yaml:146]: This line is not necessary

File: stable/rabbitmq/values.yaml [hunk 3]
```diff
@@ -139,6 +139,11 @@ service:
   annotations:
     # service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
 
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  # loadBalancerSourceRanges:
```
Comment (carrodher) [inline stable/rabbitmq/values.yaml:144]: Can you add a line only with `##` between the comment and the parameter? Check other examples:
```yaml
  ## RabbitMQ Manager port
  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
  ##
  managerPort: 15672
```

File: stable/rabbitmq/values-production.yaml [hunk 4]
```diff
@@ -139,6 +139,11 @@ service:
   annotations:
     # service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
 
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  # loadBalancerSourceRanges:
+  #  - 10.10.10.0/24
```
Comment (carrodher) [inline stable/rabbitmq/values-production.yaml:145]: Can you add a line only with `##` between the comment and the parameter? Check other examples:
```yaml
  ## RabbitMQ Manager port
  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
  ##
  managerPort: 15672
```

**Review by carrodher — CHANGES_REQUESTED** [review]
Can you take a look at the DCO comment? https://github.com/helm/charts/pull/13058/checks?check_run_id=102718624

**Comment by jmvizcainoio** [comment]
sorry but i destroy my environment and i created the pr again

https://github.com/helm/charts/pull/13081

I add a conditional to check if type is loadbalancer

**Pull request closed without merging**